src/main/msp/msp_serial.c

```diff
@@ -534,14 +562,24 @@ int mspSerialPush(uint8_t cmd, const uint8_t *data, int datalen)
 {
     int ret = 0;
 
+    //Do not output in climode
+    if(cliMode){
```

Comment:
Should be spaces between `if` and `(` as well as between `)` and the curly brace `{`